lib/src/api/engine/remote/ws/native.rs

```diff
@@ -306,7 +306,8 @@ pub(crate) fn router(
 										}
 									}
 									Err(_error) => {
-										trace!(target: LOG, "Failed to deserialise message");
+										// Unfortunately, we don't know which response failed to deserialize
```

Comment:
Does including the `error` in the log message help?

Reply:
> Does including the `error` in the log message help?

Good idea but unfortunately not, due to how serde untagged enum's work. When the deserialization error happens with one variant, serde moves on to the next and so the resulting error message is that no variants matched.

https://github.com/surrealdb/surrealdb/blob/a8f35611c222f26df81c7f82362b91fc7298e54a/lib/src/api/engine/remote/ws/mod.rs#L106C1-L118